Contest_API.md

```diff
@@ -497,45 +497,45 @@ NDJSON format.
 
 In the tables below, the columns are:
 
-  - Name: Attribute name; object sub-attributes are indicated as
-    `object.attribute`.
-  - Type: Data type of the attribute; one of the 
-    [types listed above](#json-attribute-types).
-  - Required?: Whether this is a required attribute that **must** be
+  - Name: Property name; object sub-properties are indicated as
+    `object.property`.
```

Comment:
The object would be the submission or the judgement, so this should really be `property.subproperty` not `object.subproperty`.

Also, is it clear what a "sub-property" is?

Reply:
We could mention that we're essentially using JSONPath notation. That might also clarify the meaning of sub-property.

Reply:
Is there some good documentation for JSONPath notation we could link to?

Reply:
I fixed the text to read property.subproperty in the 2nd commit, let me know the text/link if you want to include in this PR, otherwise we can add the JSONPath reference separately.

Reply:
Since this is not exactly JSONPath (that always starts with `$`) and I couldn't find a nice reference, I'm happy to leave this as is.

Reply:
I think this is good for now.